### PR TITLE
Support config for container image

### PR DESCRIPTION
**Pull request by liangyuanpeng**
## Description

Add param of keepalived-image for openelb manager
You can specify all images used by openelb through configmap. **Name must be `openelb-images`.**
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: openelb-images
  namespace: openelb-system
data:
  forward-image: kubesphere/openelb-forward:master
  proxy-image: kubesphere/openelb-proxy:master
  keepalived-vip-image: kubesphere/kube-keepalived-vip:0.35
```
## Related links:

#283  https://github.com/kubesphere/openelb/issues/214

### CODE REVIEW

**Comment by renyunkang** [comment]
we don't have the image `kubesphere/openelb-forward:v0.5.0` and `kubesphere/openelb-proxy:v0.5.0`.

**Comment by renyunkang** [comment]
kindly ping @liangyuanpeng

**Comment by liangyuanpeng** [comment]
> How about specifying the keepalived image through ConfigMap, like openelb-proxy and openelb-forward, it is convenient for us to manage it uniformly.

make sense,  let me update it.

**Comment by renyunkang** [comment]
**Thank you very much for your contribution**. I was wondering if we could use a configmap to specify all the custom images used by openelb. For example: configmap `openelb-image-configmap` defines `forward-image/proxy-image/keepalived-vip-image` and other images used by openelb. In addition, whether we still need to provide command line parameters to specify the `image` after specifying the images through configmap.

File: pkg/speaker/vip/keepalived.go [hunk 1]
```diff
@@ -106,6 +105,22 @@ func (k *KeepAlived) Start(stopCh <-chan struct{}) error {
...
+	return k.clientset.CoreV1().ConfigMaps(util.EnvNamespace()).
+		Get(context.Background(), constant.OpenELBImagesConfigMap, metav1.GetOptions{})
+}
+
+func (k *KeepAlived) getImage() string {
+	cm, err := k.getConfig()
+	if err != nil {
+		return constant.OpenELBDefaultKeepAliveImage
+	}
+	return cm.Data[constant.OpenELBKeepAliveImage]
```
Comment (chaunceyjiang) [inline pkg/speaker/vip/keepalived.go:121]: `cm.Data[constant.OpenELBKeepAliveImage]`
Is it possible for null pointer to occur here?

Comment (renyunkang) [reply pkg/speaker/vip/keepalived.go]: oh, may be empty

**Comment by chaunceyjiang** [comment]
Thanks!

/lgtm

**Comment by liangyuanpeng** [comment]
Thanks for your work.

/lgtm

> whether we still need to provide command line parameters to specify the image after specifying the images through configmap?

Some Suggest ,Only configmap or some config file should be enought.

**Pull request merged**